release_notes/release_notes.stex

```diff
@@ -58,6 +58,12 @@ Online versions of both books can be found at
 %-----------------------------------------------------------------------------
 \section{Functionality Changes}\label{section:functionality}
 
+\subsection{Process environment setting (non WIN32) (9.5.3)}
```

Comment:
I think this may better be characterized as a bug fix, because you didn't change what putenv does.
\subsection{\protect\code{putenv} memory leak (9.5.3)}